text_to_video/docker/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM pytorch/pytorch:2.5.1-cuda12.1-cudnn9-devel
```

Comment:
Seems like this is pre-Blackwell, recommend adding the supported hardware in the README